rasa_core/policies/ensemble.py

```diff
@@ -206,10 +206,15 @@ def probabilities_using_best_policy(self, tracker, domain):
         # type: (DialogueStateTracker, Domain) -> List[float]
         result = None
         max_confidence = -1
-        for p in self.policies:
+        best_policy_name = ''
+        for i, p in enumerate(self.policies):
             probabilities = p.predict_action_probabilities(tracker, domain)
             confidence = np.max(probabilities)
             if confidence > max_confidence:
                 max_confidence = confidence
                 result = probabilities
+                best_policy_name = 'policy_{}_{}'.format(i, type(p).__name__)
```

Comment:
in case we have several policies with the same type, like Keras with different featurizers. It is how we name the folders in persist, so i wanted logger to be consistent with that.